monitoring/README.md

```diff
@@ -157,7 +157,7 @@ At this point, all three services should be set up: the monitoring server on the
 
 1. Follow [these instructions]() to install and run Grafana on the VM. The instructions are for Debian/Ubuntu, since this is the OS which the VM runs.
 
-2. To access the Grafana dashboard from your local computer, make sure you are connected to the VPN for the switch (you should already have openvpn set up by this point). Open a browser and navigate to your switch address: for example, `172.31.x.254:3000`, where x is your group number. The port is 3000 since this is Grafana's default server port. 
+2. To access the Grafana dashboard from your local computer, make sure you are connected to the VPN for the switch (you should already have openvpn set up by this point). Open a browser and navigate to your switch address: for example, `172.31.x.254:3000`, where x is your group number. The port is 3000 since this is Grafana's default server port.
```

Comment:
it's not your switch address. it's the piman address right?

Reply:
Oh right I made a change to that